auto-run-nondex/valid_repo_extractor.py

```diff
@@ -6,7 +6,10 @@
 AUTHOR_URL_POSTFIX = 'language=java&q=&sort=&type='
 AUTHOR_URL = ''
 REPO_URL_PREFIX = 'https://github.com'
-PR_DATA_CSV_FILE_URL = 'https://raw.githubusercontent.com/TestingResearchIllinois/idoft/main/pr-data.csv'
+PR_DATA_CSV_FILE_URL = (
```

Comment:
(How) can `flake8` be configured to allow longer lines?

Reply:
I've skipped the long line checks for now, I can configure it and allow longer lines if needed